cmd/copy.go

```diff
@@ -272,6 +273,24 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 	}
 
 	cooked.fromTo = fromTo
+
+	// Get file path from user which would contain list of all versionIDs
+	// Process the file line by line and then prepare a list of all version ids of the blob.
+	if raw.listOfVersionIDs != "" {
+		filePtr, err := os.Open(raw.listOfVersionIDs)
+		if err != nil {
+			return cooked, err
+		}
+		defer filePtr.Close()
+		scanner := bufio.NewScanner(filePtr)
+		scanner.Split(bufio.ScanLines)
+		var versionIDs []string
+		for scanner.Scan() {
+			versionIDs = append(versionIDs, scanner.Text())
+		}
+		cooked.listOfVersionIDs = versionIDs
+	}
```

Comment:
please follow the same workflow as the list-of-files, i.e. not read out the entire file right away, but pass a channel to the traverser. Also, we need to skip empty lines, and UTFBOM, both can become sources of bugs.

Reply:
Done